app/components/search/SearchResults/SearchResults.jsx

```diff
@@ -194,7 +161,7 @@ const SearchResult = ({ hit, index, setCenterCoords }) => {
           <Link to={`/organizations/${resourceId}`}>{hit.service_of}</Link>
         </div>
         <div className={styles.address}>{renderAddressMetadata(hit)}</div>
-        <ReactMarkdown className={`rendered-markdown ${styles.description}`} source={hit.long_description} />
+        <ReactMarkdown className={`rendered-markdown ${styles.description}`} source={hit.long_description} linkTarget="_blank" />
```

Comment:
Adding the linkTarget="_blank" attr to the ReactMarkdown component adds target="_blank" to all links in the markdown; this is for the service description